src/cursor.rs

```diff
@@ -394,13 +399,13 @@ pub enum SetCursorStyle {
 impl Command for SetCursorStyle {
     fn write_ansi(&self, f: &mut impl fmt::Write) -> fmt::Result {
         match self {
-            SetCursorStyle::DefaultUserShape => f.write_str("\x1b[?12h\x1b[0 q"),
```

Comment:
I was thinking perhaps a little redundancy could not hurt. But actually since we do have Enable/DisableBlinking it is a bit uneasily.